src/components.js

```diff
@@ -826,14 +827,14 @@ const components = [
     name: 'Fixed',
     type: 'div',
     props: {},
-    style: props => ({
+    style: props => Object.assign({
       position: 'fixed',
       top: props.top ? 0 : null,
       right: props.right ? 0 : null,
       bottom: props.bottom ? 0 : null,
       left: props.left ? 0 : null,
       zIndex: props.z
-    }),
+    }, center(props)),
```

Comment:
I have thought about it and chosen this approach because of the `top`, `left`, `z-index` and other have own logic, they may be set or not. On the other hand centering have own logic with `transform`. If we place all this logic in one function it would be quite complicated. Therefore `center` function responsible only for centering. If need to extract remaining part of logic from `components.js` it would be placed in a separate function like `position` that will have the invocation of `center` function.